tempest-testing-internal/src/main/kotlin/app/cash/tempest/testing/internal/TestUtils.kt

```diff
@@ -32,6 +32,10 @@ import java.net.InetSocketAddress
 import java.net.ServerSocket
 import java.net.Socket
 
+fun pickRandomPort(): Int {
```

Comment:
Bringing this back because it was a public method I had unintentionally deleted in the previous PR.